website/conferences/utils.py

```diff
@@ -41,7 +41,7 @@ def provision_node(conference, message, node, user):
     if not message.is_spam and conference.public_projects:
         node.set_privacy('public', meeting_creation=True, auth=auth)
 
-    node.add_tag(message.conference_name, auth=auth)
+    conference.submissions.add(node)
```

Comment:
Instead of adding a tag to the meeting submission, this links conferences and nodes in the db.  The main issue is that with this change, users can only submit to a conference through the workflow.  (We can discuss areas to surface this if need-be).